### DISCUSSION TITLE
世界盃與AI的結合

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#566] [General]
太猛了吧!今年卡達世足賽賽事場館的即時監測運作系統，運用AI人臉辨識、數據人流監控，即時監測避免像是恐佈攻擊衝突、資安問題等事件發生。其他國家應該向他們多多學習才對!
https://udn.com/news/story/7240/6826040

**Comment by awehao** [comment]
算是解決了安全隱患呢，球員可以更加專注於比賽

**Comment by Chenszuhong** [comment]
真的很帥 進球或是出界都可以用科技去判斷說是否成立

**Comment by Aerofe** [comment]
這些科技讓觀眾更加享受比賽，也不必擔心安全問題，這些科技開始普及的話會帶來更多的益處

**Comment by Raywei0214** [comment]
雖然科技很厲害，但它害我輸錢了我一點也不高興，期許未來能夠有更多應用，匹如測速照相升級，好幾次我都超速，結果都沒拍。

**Comment by magicmicheal** [comment]
有了這些科技幫助，越位就是越位，手摸到球就是摸到球，卻實大大的增加了公平性，但也就缺少了，裁判也是比賽的一部份，我沒有認為完全公平沒有什麼不好，但就會喪失了一些比賽的不可預測性，最有名的例子自然是馬拉度納在世足上的那次手球，幫助阿根廷逆轉拿下冠軍，若是有了AI的幫助，這歷史上的一刻，便不存在

**Comment by bww0401** [comment]
還記得前幾屆在越位的判定上，會時常出現爭議，但這屆多了AI幫助，不僅有效給出即時判定，亦可以增加判定的正確性，讓裁判可以把更多眼力與心力放在球員的犯規上!

**Comment by titan92601** [comment]
ai雖然可以方便我們去判斷這些問題，但能否用在所有的比賽就要再加以深思，畢盡許多比賽的勝負被齁牽扯了許多關係，ai能否以公平的角度去判別，或是在比賽前就被他人動手腳改被改其公正性，這些都要加以深思

**Comment by manipulateed** [comment]
AI將會跟更多產業結合，可能是前幾年的股市真的太慘烈，從現在這個復甦的時機點，加上AI的發光發熱，在AI的投資的確是該被看好而下去做投資。
